Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift

```diff
@@ -164,8 +161,6 @@ public enum FeatureFlag: String, CaseIterable {
             false
         case .endOfYear:
             false
-        case .newShowNotesEndpoint:
-            false
```

Comment:
@danielebogo Just realized that we cannot remove the remote FF from Firebase, because of the above.
The default value is set to false so the older version where this was activated because of the remote FF will have the functionality disabled because the default value on the code is a false.
Am I missing something?